libraries/ts-command-line/src/CommandLineParameter.ts

```diff
@@ -582,12 +591,32 @@ export class CommandLineStringListParameter extends CommandLineParameterWithArgu
         // NOTE: If the environment variable is defined as an empty string,
         // here we will accept the empty string as our value.  (For number/flag we don't do that.)
 
-        // In the current implementation, the environment variable for a "string list" can only
-        // store a single item.  If we wanted to allow multiple items (and still have a conventional-seeming
-        // environment), we would ask the caller to provide an appropriate delimiter.  Getting involved
-        // with escaping here seems unwise, since there are so many shell escaping mechanisms that could
-        // potentially confuse the experience.
-        this._values = [ environmentValue ];
+        if (environmentValue[0] === '[') {
```

Comment:
Yes, but then it will not be parsed as an array. The entire string will simply become the first item in the list.  

Hmm... that might be counterintuitive if a person is typing the value into a text box in a web page (e.g. AzureDev Ops).  I'll trim it. 👍

Reply:
I also documented the parsing rules.